src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
   margin: 0;
+  display: flex;
```

Comment:
Properties within selectors should be ordered consistently. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), properties should be grouped by type and ordered within those groups in a specific way. For example, positioning properties (`position`, `top`, `right`, `bottom`, `left`, `z-index`) should come before box model properties (`display`, `flex-direction`, etc.).